radio/src/datastructs.h

```diff
@@ -954,6 +954,7 @@ PACK(struct TrainerData {
 #elif defined(PCBSKY9X)
   #define EXTRA_GENERAL_FIELDS \
     EXTRA_GENERAL_FIELDS_ARM \
+    NOBACKUP(uint8_t  serial2Mode:4); \
```

Comment:
ouch, really? you break all existing SKY9X / 9XR-PRO EEPROM with this line!

Reply:
Also if eeprom is changed, we need changes to companion as well